src/python/bot/tasks/minimize_task.py

```diff
@@ -1353,19 +1351,26 @@ def handle_unreproducible():
     if cleansed_testcase_path:
       current_testcase_path = cleansed_testcase_path
 
+  # Store the minimized testcase.
+  testcase = data_handler.get_testcase_by_id(testcase_id)
+  with open(current_testcase_path, 'rb') as file_handle:
+    minimized_keys = blobs.write_blob(file_handle)
```

Comment:
This avoids creating 6 blob entries instead of 1.

Reply:
From memory this was intentional, so we can see minimize results earlier while task is still running. No strong opinions about keeping this though.